ggml.c

```diff
@@ -8187,7 +8189,7 @@ static void ggml_compute_forward_rms_norm(
 
 // ggml_compute_forward_mul_mat
 
-#if defined(GGML_USE_ACCELERATE) || defined(GGML_USE_OPENBLAS) || defined(GGML_USE_CLBLAST)
+#if defined(GGML_USE_ACCELERATE) || defined(GGML_USE_OPENBLAS) || defined(GGML_USE_CLBLAST) || defined(GGML_USE_CLBLASTNETLIB)
```

Comment:
Let's have this build define both `GGML_USE_CLBLAST` and `GGML_USE_CLBLASTNETLIB` and avoid these changes

Reply:
You are correct, I get the logic.